src/main/java/constants/Command.java

```diff
@@ -7,6 +7,7 @@ public class Command {
     public static final String VIEW_COMMAND = "view";
     public static final String DELETE_COMMAND = "delete";
     public static final String SALES_COMMAND = "sales";
+    public static final String SCHEDULE_COMMAND = "schedule";
```

Comment:
I like the use of the work `schedule`. It allows for separation of concerns between the different user commands of our app (`catalog`, `schedule`, and `sales`). It allows allows us to include more schedule-related features in the future (other than setting a deadline).